biocypher_metta/adapters/ontologies_adapter.py

```diff
@@ -70,13 +70,11 @@ def get_nodes(self):
             term_id = OntologyAdapter.to_key(node)
             # 'uri': str(node),
             term_name = ', '.join(self.get_all_property_values_from_node(node, 'term_names'))
```

Comment:
Can you add option to specify whether to include description or not (e.g `add_description`)? We might need to add the descriptions some time later. This option would be passed in the `adapters_config.yaml` and `adapters_config_sample.yaml`.